### DISCUSSION TITLE
Generating anndata from Cell Ranger output on Multiome dataset

### DISCUSSION THREAD

**Discussion by sylestiel** [theislab/cellrank#831] [Q&A]
Hi ,
If I have Cellranger derived possorted_bam.bam file on a 10X Multiome Dataset how may I use that to arrive at anndata for CellRank analysis. 
Can you please provide the script to do so.

Thanks!!!

**Comment by Marius1311** [comment]
I think that question is not really about CellRank, but more broadly about how to create AnnData objects from CellRanger output files. Please check the AnnData/scanpy tutorials for this and potentially open an issue there / look through open and closed issues.
